File: C1.UWP.FlexGrid/CS/FlexGridSamples/Data/CountryInitialConverter.cs
using System;
using System.Collections.Specialized;
using Windows.UI.Xaml.Data;

namespace FlexGridSamples
{
    // converter used to group countries by their first initial
    class CountryInitialConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string culture)
        {
            return ((string)value)[0].ToString().ToUpper();
        }
        public object ConvertBack(object value, Type targetType, object parameter, string culture)
        {
            throw new NotImplementedException();
        }
    }
}

File: C1.UWP.FlexGrid/VB/FlexGridSamples/Data/CountryInitialConverter.vb
Imports Windows.UI.Xaml.Data
Imports System.Collections.Specialized
Imports System

' converter used to group countries by their first initial
Class CountryInitialConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, culture As String) As Object Implements IValueConverter.Convert
        Return (CType(value, String))(0).ToString().ToUpper()
    End Function
    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, culture As String) As Object Implements IValueConverter.ConvertBack
        Throw New NotImplementedException()
    End Function
End Class